src/meteodatalab/data_source.py

```diff
@@ -23,17 +23,14 @@
 @contextmanager
 def cosmo_grib_defs():
     """Enable COSMO GRIB definitions."""
-    if "ECCODES_DEFINITION_PATH" in os.environ or "GRIB_DEFINITION_PATH" in os.environ:
-        return nullcontext()
-
     restore = eccodes.codes_definition_path()
     root_dir = Path(sys.prefix) / "share"
     paths = (
         root_dir / "eccodes-cosmo-resources/definitions",
         Path(restore),
     )
     for path in paths:
-        if not path.exists():
+        if not path.exists() and not str(path).startswith("/MEMFS"):
```

Comment:
is it tested? 
Does this work?
```
eccodes.codes_set_definitions_path('/real/path/:/MEMFS')
```

Reply:
I tried it and it works but it's just setting a string. I believe that the part that was not working is that it's not an actual directory on disk but this should be addressed.